sparkbox/gem-me-bot
# Hotfix: Fix issue posting Gems to channels

## Descriptions
In February 2021, the `channels.history` API was deprecated in favor of the `conversations.history` API. To utilize the new API, we need to update our Slack and Slackbot dependencies. This PR updates both dependencies and utilizes the new API.

## Validation Steps
1. Pull down this branch.
1. Copy the tokens from the Heroku instance into your local `.env`.
1. Apply the [supplied patch](https://github.com/sparkbox/gem-me-bot/files/6092133/patch_hotfix--update-dependencies.zip).
2. Run `npm i`.
3. Run `npm start`.
4. While the app is running, find something gem-worthy in Slack and gem it.
- [ ] Confirm the gem is posted to the gems testing channel.
